_homework/interaction-design/sketch-symbols.md

```diff
@@ -4,17 +4,43 @@ assigned: Aug 31, 2018
 due: Sept 4, 2018
 ---
 
-**Note:** More details coming soon.
+Now that you have built some cards and changed photos, descriptions and titles using nested symbols, we'd like for you to see how else you can implement them in your designs! 
 
+**Upload your final Sketch files** [to the Google Drive](https://drive.google.com/drive/u/1/folders/1OJWSXlXKHCHchXesu9GJhFflCIS-RvV0) in the "Completed Work" folder.
 
-Sketch Tutorials
-----------------
 
+1st Deliverable: Design a Button System
+-------------------------------------------------
 
-Exercise
---------
+1. Watch [Creating a Button System with Nested Symbols on Sketch App](https://www.youtube.com/watch?v=_bjqVF7Fvg4&t=794s) from Sketch Together <small>Duration: 30:39</small>
```

Comment:
Looks like this link takes you to a particular time in the video. Is that intentional?

If not, you probably want https://www.youtube.com/watch?v=_bjqVF7Fvg4

Reply:
Oh no not intentional. Good catch! Will change it.